dune/gdt/local/operators/advection-dg.hh

```diff
@@ -87,7 +88,9 @@ public:
     const auto local_source = source.local_discrete_function(element);
     const auto local_source_order = local_source->order(param);
     const auto local_basis_order = basis.order(param);
-    const auto integrand_order = flux_.order(param) * local_source_order + std::max(local_basis_order - 1, 0);
+    const auto local_flux = flux_.local_function();
```

Comment:
this `local_flux_` could/should be a mutable member

Reply:
Yes, I already changed that in my second pull request. I also have some changes to the interface such that the ``local_source`` is also a mutable member. Obviously, this requires that the source is passed in the constructor, but that is not a problem for all local operators that are implemented by now. I am almost finished with that PR, too, we can then have this discussion over there :).